docs/NETWORKING.md

```diff
@@ -0,0 +1,36 @@
+# Networking
+
+All testground runners _except_ for the local runner have two networks: a
+control and a data network.
+
+* Test instances communicate with each other over the data network.
+* Test instances communicate with the sync service, and _only_ the sync service,
+  over the control network.
+  
+The local runner will use your machines local network interfaces.
+
+## Control Network
+
+The "control network" runs on 192.168.0.1/16 and should only be used to
```

Comment:
@Stebalien could you also update `infra/docker-swarm/ansible/roles/testground-services/tasks/main.yaml` with the IPAM, when you are about to merge this?